libraries/node-core-library/src/Terminal/FileTerminalProvider.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { ITerminalProvider } from './ITerminalProvider';
+import { FileSystem } from '../FileSystem';
+
+/**
+ * @beta
+ */
+export class FileTerminalProvider implements ITerminalProvider {
```

Comment:
>FileTerminalProvider [](start = 13, length = 20)

If you're going to provide standard ITerminalProvider instances for people, there's a bunch of configuration options that probably would be needed for the different scenarios.

The implementations here are so simple that it might be easier for consumers to simply write their own providers, than to try to configure a generic one.